docs/source/blogs/index.rst

```diff
@@ -0,0 +1,7 @@
+Blog Posts
+==========
+
+.. toctree::
+   :maxdepth: 1
+
+   [Nov. 2019] Streamlining Variant Normalization on Large Genomic Datasets <variant-normalization-blog/variant-normalization-blog>
```

Comment:
Do we need to put this link explicitly? Typically we let the toctree directive list all the blogs.

Reply:
This is to have the date of the blog in toc but not in the title line of the blog itself.